composer.json

```diff
@@ -20,7 +20,7 @@
 
     "autoload": {
         "psr-4": {
-            "WhichBrowser\\": [ "src/", "tests/src/" ]
+            "WhichBrowser\\": [ "src/" ]
```

Comment:
You can probably maintain the current behavior by adding
```
            "WhichBrowser\\": [ "tests/src/" ]
```
to autoload-dev (near line 43 below).

Reply:
Yup, 77a681d7bc20d54212e5b343ae1bb5dfa1594c72 fixes it.